A test failure... how is this related to my change?

```
[Test teensy35]Teensy3.5 with default config...In file included from Marlin/src/module/stepper_indirection.cpp:38:0:
Marlin/src/module/../module/stepper.h: In function 'void reset_stepper_drivers()':
Marlin/src/module/../module/stepper.h:475:17: error: 'static void Stepper::set_directions()' is private
static void set_directions();
^
Marlin/src/module/stepper_indirection.cpp:585:11: error: within this context
stepper.set_directions();
^
In file included from Marlin/src/module/stepper_indirection.cpp:38:0:
Marlin/src/module/../module/stepper.h:475:17: error: 'static void Stepper::set_directions()' is private
static void set_directions();
^
Marlin/src/module/stepper_indirection.cpp:585:26: error: within this context
stepper.set_directions();
^
*** [.pioenvs/teensy35/src/src/module/stepper_indirection.cpp.o] Error 1
```

--- a/Marlin/src/lcd/ultralcd.cpp
+++ b/Marlin/src/lcd/ultralcd.cpp
@@ -2846,7 +2846,7 @@ void lcd_quick_feedback(const bool clear_buttons) {
       MENU_BACK(MSG_MAIN);
       #if ENABLED(DELTA_AUTO_CALIBRATION)
         MENU_ITEM(gcode, MSG_DELTA_AUTO_CALIBRATE, PSTR("G33"));
-        MENU_ITEM(gcode, MSG_DELTA_HEIGHT_CALIBRATE, PSTR("G33 P1"));
+        MENU_ITEM(gcode, MSG_DELTA_HEIGHT_CALIBRATE, PSTR("G33 S P1"));
         MENU_ITEM(gcode, MSG_DELTA_Z_OFFSET_CALIBRATE, PSTR("G33 P-1"));
         #if ENABLED(EEPROM_SETTINGS)
           MENU_ITEM(function, MSG_STORE_EEPROM, lcd_store_settings);

